src/parsers/urdf/geometry.hxx

```diff
@@ -93,7 +111,13 @@ namespace pinocchio
         mesh->scale.z;
       }
 
-#ifdef PINOCCHIO_WITH_HPP_FCL      
+#ifdef PINOCCHIO_WITH_HPP_FCL
+# if ( HPP_FCL_MAJOR_VERSION>1 || ( HPP_FCL_MAJOR_VERSION==1 && \
+      ( HPP_FCL_MINOR_VERSION>1 || ( HPP_FCL_MINOR_VERSION==1 && \
+                                     HPP_FCL_PATCH_VERSION>3))))
+#  define PINOCCHIO_HPP_FCL_SUPERIOR_TO_1_1_3
```

Comment:
We usually say `GREATER` here

Reply:
Is this really important ? This variable is only used locally and its meaning is clear.

Reply:
It's not :)